docs/docs-content/clusters/public-cloud/azure/azure-disk-encryption.md

```diff
@@ -0,0 +1,353 @@
+[validating the disk encryption enablement](#validate-after-enablement).
+
+## Disable Disk Encryption
+
+### Disable Disk Encryption on a Cluster Profile
+
+Use the following steps to disable disk encryption on an
+[existing cluster profile](../../../profiles/cluster-profiles/modify-cluster-profiles/modify-cluster-profiles.md).
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+2. Ensure you are in the correct project scope.
+
+3. From the left **Main Menu**, select **Profiles** and click the cluster profile that you want to edit.
+
+4. Create a new version of your cluster profile:
+
+   1. Click the version drop-down menu next to the cluster profile name, and click **Create new version**.
+   2. Fill the **Version** field with a new version number.
+   3. Click **Confirm**.
+
+5. Select the Kubernetes layer to view the **Edit Pack** drawer.
+
+6. In **Values**, click the **\</\>** button to show the YAML editor.
+
+7. On the right-hand side, click the **Presets** drop-down menu, and select the **Disable Encryption Using
+   Customer-Managed Key** option.
+
+8. Click **Confirm updates**.
+
+9. Click **Save Changes**.
+
+### Disable Disk Encryption on an Active Cluster
+
+Use the following steps to disable disk encryption on an active cluster using the
+[updated cluster profile](#disable-disk-encryption-on-a-cluster-profile).
+
+:::warning
+
+Performing these steps will cause a
+[full cluster repave](../../cluster-management/node-pool.md#repave-behavior-and-configuration).
+
+:::
+
+1. From the left **Main Menu**, select **Clusters**.
+
+2. Find the cluster that you want to update and click on it.
+
+3. Click the **Profile** tab.
+
+4. Click the version drop-down menu in **Infrastructure Layers** and select the version that has disk encryption
+   disabled.
+
+5. Click **Review & Save**, then click **Review changes in Editor** in the Changes Summary box.
+
+6. Review the changes and click **Update**.
+
+Your cluster will now update and a full cluster repave will occur. Wait until the update has completed before
+[validating the disk encryption disablement](#validate-after-disablement).
+
+## Validate
+
+### Validate after Enablement
+
+Follow these steps to validate enablement of customer-managed key encryption on your Azure VM disks.
+
+1. Log in to the [Azure Portal](https://portal.azure.com/).
+
+2. In the search bar, look for **Disks**. Click on the service when found.
+
+3. Find your cluster's disks by using the search filters provided. You can use the cluster name as the cluster resources
+   contain the cluster name at the beginning (for example: `<clusterName>-e3c0-f7ljd_OSDisk`).
+
+4. Once identified, click on a disk name to view its details.
+
+5. Scroll down to view the **Properties** tab and check that the **Encryption** section shows **Customer-managed key**
+   for **Encryption type**. Your Disk Encryption Set name is also shown for the **Encryption key**.
+
+6. Repeat steps 4-5 for each disk in your cluster.
```

Comment:
```suggestion
6. Repeat steps four and five for each disk in your cluster.
```

Reply:
```suggestion
6. Repeat steps 4 and 5 for each disk in your cluster.
```

Reply:
@caroldelwing In our style-guide, we stipulate:

> In how-to steps, if instructions refer to a specific step or steps, use numbers instead of spelling out the word.